clang/lib/StaticAnalyzer/Core/BugReporter.cpp

```diff
@@ -2827,6 +2827,12 @@ void BugReporter::emitReport(std::unique_ptr<BugReport> R) {
   if (!ValidSourceLoc)
     return;
 
+  // The goal of system headers is that no diagnostic report is emitted in
```

Comment:
Well, that is not the goal of system headers :) What about `Do not emit a bug report if it is located in a system header.` as a comment?